client/my-sites/plugins/plugin-autoupdate-toggle/index.jsx

```diff
@@ -16,12 +16,17 @@ module.exports = React.createClass( {
 	displayName: 'PluginAutopdateToggle',
 
 	propTypes: {
+		isMock: React.PropTypes.bool,
 		site: React.PropTypes.object.isRequired,
 		plugin: React.PropTypes.object.isRequired,
 		wporg: React.PropTypes.bool
 	},
 
 	toggleAutoupdates: function() {
+		if( this.props.isMock ) {
```

Comment:
yeah, I don't even need it, if I don't pass the mock site, it would not show this toggles... but if the point of the mocked-components is to show how the feature would work if active, I guess the interactive part is the most interesting one to show